apps/spruce/src/components/DayPicker/index.tsx

```diff
@@ -50,6 +52,7 @@ export const DayPicker: React.FC<{
       {days.map((day, i) => (
         <Day
           day={day}
+          disabled={!!disabled}
```

Comment:
Could we set `disabled = false` in the function props so that we can pass `disabled={disabled}`?